server/channels/store/storetest/retention_policy_store.go

```diff
@@ -492,7 +492,7 @@ func testRetentionPolicyStoreRemoveChannels(t *testing.T, ss store.Store, s SqlS
 		err := ss.RetentionPolicy().RemoveChannels(policy.ID, []string{channelID})
 		require.NoError(t, err)
 		// verify that the channel was actually removed
-		copy := copyRetentionPolicyWithTeamAndChannelIds(policy)
+		copy := copyRetentionPolicyWithTeamAndChannelIds(policy) //nolint:revive
```

Comment:
Why not `s/copy/c` here like above?

Reply:
I did that, and then realized it was becoming unreadable. I'll revert the earlier change.